### DISCUSSION TITLE
Automation Ableton 10

### DISCUSSION THREAD

**Discussion by inspektormarek** [apohl79/audiogridder#607] [Help]
Hi,

first of all your Product is amazing!

i can’t get my automation to work in Ableton 10. and im wondering how you guys manage it. 

I open the the AG Plugin in Ableton there I go to configure  
![image](https://user-images.githubusercontent.com/85619873/121344657-4f37e180-c924-11eb-8db2-62d13d969f0f.jpg)

and then I open the generic Editor but if I press a settings it doesn’t connect to the plugin. Did I something wrong?

**Comment by inspektormarek** [comment]
Okay nevermind i solved it!

**Reply by DJUnit27** [reply]
How did you go about it?

**Reply by stubar** [reply]
I would also like to do this, please share solution.
